src/test/java/org/folio/service/processing/ParallelFileChunkingProcessorUnitTest.java

```diff
@@ -14,10 +14,7 @@
 import org.folio.kafka.KafkaConfig;
 import org.folio.kafka.KafkaTopicNameHelper;
 import org.folio.rest.AbstractRestTest;
-import org.folio.rest.jaxrs.model.Event;
-import org.folio.rest.jaxrs.model.FileDefinition;
-import org.folio.rest.jaxrs.model.JobProfileInfo;
-import org.folio.rest.jaxrs.model.RawRecordsDto;
+import org.folio.rest.jaxrs.model.*;
```

Comment:
I forgot to mention this line. Please remove wildcard import from tests too.

Reply:
updated.